images/php-cli/entrypoints/61-php-xdebug-cli-env.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh
 
-# enable XDebug only if XDEBUG_ENABLE is set to true or TRUE or True
-if expr "$XDEBUG_ENABLE" : '[Tt][Rr][Uu][Ee]' > /dev/null; then
-  # XDEBUG_CONFIG is used by xdebug to decide if an xdebug session should be started in the CLI or not.
+# Only if XDEBUG_ENABLE is not empty
+if [ ! -z ${XDEBUG_ENABLE} ]; then
+  # XDEBUG_SESSION is used by xdebug to decide if an xdebug session should be started in the CLI or not.
```

Comment:
This includes a regression of #251 which isn't in #41, so I believe this was just a merge conflict resolution error?

Reply:
Agreed. I forced pushed to the branch to fix this.